src/Libraries/CoreNodeModels/Enum.cs

```diff
@@ -61,7 +69,7 @@ protected AllChildrenOfType() : base("Types")
         }
 
         [JsonConstructor]
-        protected AllChildrenOfType(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts)
+        protected AllChildrenOfType(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base("Types", inPorts, outPorts)
```

Comment:
I think where possible we should start pulling strings out of the constructors and creating constants we can reference.